blocks/header/header.css

```diff
@@ -183,7 +114,142 @@ header nav .nav-sections ul > li > ul > li {
   font-weight: var(--mt-font-weight-bold);
 }
 
-@media (min-width: 900px) {
+.icon-logo {
+  width: 200px;
+}
+
+
+
+@media (max-width: 1024px) {
```

Comment:
should try to eliminate max-width breakpoints and override where necessary in a min-width break-point